invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -177,21 +177,24 @@
             },
             "locations" : {
               "properties" : {
-                "description" : {
-                  "type" : "text"
-                },
-                "place" : {
-                  "type" : "text"
-                },
-                "point" : {
+                "geometry": {
                   "properties" : {
-                    "lat" : {
-                      "type" : "double"
+                    "type" : {
+                      "type" : "keyword"
                     },
-                    "lon" : {
-                      "type" : "double"
+                    "coordinates" : {
+                      "type" : "float"
```

Comment:
Minor: If we start with an assumption that we can only accept Point objects, then type here should be ``geo_point`` (https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html)

For later:
Later to deal with more than points we would have to extend with ``geo_shape`` (https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html) but that would require we develop a dumper that's able to split them correctly into separate indexing properties.

Reply:
I would keep it as `float` for now. Mainly because I am worried about:

> Please note that string geo-points are ordered as lat,lon, while array geo-points are ordered as the reverse: lon,lat.
> Originally, lat,lon was used for both array and string, but the array format was changed early on to conform to the format used by GeoJSON.

Also the input will be `[float, float]`, but it will output `{lon: float, lat: float}`. We would need to account for that at (de)serialization. 

I would tackle this when doing the dumper. WDYT?

Reply:
I think it's mainly the ES loading that's the issue (but don't we have the original source in ``_source``). Can you log an issue and assign to November board. Thanks.

Reply:
The order (lon,lat) should be fine, as we have specified that we accept GeoJSON which is in that order.

Reply:
Done https://github.com/inveniosoftware/invenio-rdm-records/issues/265